studio/pages/new/[slug].tsx

```diff
@@ -147,6 +147,14 @@ const Wizard: NextPageWithLayout = () => {
     setDbPricingTierKey(value)
   }
 
+  function onPaymentMethodAdded() {
+    console.log({ slug })
+    console.log('pm added')
```

Comment:
Looks like some cheeky `console.log()` calls snuck in

Reply:
Whoopsie